Docs/user_guide/coordinate_systems.md

```diff
@@ -0,0 +1,268 @@
+called **IJKtoLPS**- or **IJKtoRAS**-matrix, because it represents the
+transformation from IJK to LPS or RAS.
+
+## 2D example or calculating an *IJtoLS*-matrix
+
+The following figure shows the anatomical space with an L(P)S basis on the
+left and the corresponding image coordinates on the right.
+
+![IJtoLS](https://github.com/Slicer/Slicer/releases/download/docs-resources/IJtoLS.png){.bg-warning w=550px align=center}
+
+The origin (the coordinates of the first pixel in anatomical space) is
+(50, 300) mm and the spacing (the distance between two pixels) is
+(50, 50) mm.
+
+As this is a 2D example $\mathbf{A}$ is a $2 \times 2$ matrix and $\vec{t}$ a
+$2 \times 1$ vector. Therefore, the equation of the affine transformation is:
+
+$$
+\begin{pmatrix} L \\ S \\ 1 \end{pmatrix} =
+\begin{pmatrix} A_{11} & A_{12} & t_1 \\ A_{21} & A_{22} & t_2 \\ 0 & 0 & 1 \end{pmatrix}
+\begin{pmatrix} i \\ j \\ 1 \end{pmatrix}
+$$
+
+By multiplying the **IJtoLS**-matrix and the vector of the right side, the
+following product will be obtained:
+
+![matrix_multiplication](https://github.com/Slicer/Slicer/releases/download/docs-resources/matrix_multiplication.png){.bg-warning align=center}
+
+The last equation and the matrix product show that a total of 6 unknown
+variables $(A_{11}, A_{12}, A_{21}, A_{22}, t_1, t_2)$ have to be determined.
+The knowledge of origin and spacing however allows the following relations
+between image and anatomical space:
+
+$$
+\begin{pmatrix} L \\ S \end{pmatrix} \equiv \begin{pmatrix} i \\ j \end{pmatrix} \qquad
+\begin{pmatrix} 50 \\ 300 \end{pmatrix} \equiv \begin{pmatrix} 0 \\ 0 \end{pmatrix} \qquad
+\begin{pmatrix} 100 \\ 300 \end{pmatrix} \equiv \begin{pmatrix} 1 \\ 0 \end{pmatrix} \qquad
+\begin{pmatrix} 50 \\ 250 \end{pmatrix} \equiv \begin{pmatrix} 0 \\ 1 \end{pmatrix} \qquad \dots
+$$
+
+Thus, at least six equations can be derived:
+
+$$
+50 &= A_{11} \cdot 0 + A_{12} \cdot 0 + t_1 \cdot 1
+300 &= A_{21} \cdot 0 + A_{22} \cdot 0 + t_2 \cdot 1
+100 &= A_{11} \cdot 1 + A_{12} \cdot 0 + t_1 \cdot 1
+300 &= A_{21} \cdot 1 + A_{22} \cdot 0 + t_2 \cdot 1
+50 &= A_{11} \cdot 0 + A_{12} \cdot 1 + t_1 \cdot 1
+250 &= A_{21} \cdot 0 + A_{22} \cdot 1 + t_2 \cdot 1
+$$
+
+As mentioned above, the translation $\vec{t}$ contains the information about
+the geometric position of the first pixel and is therefore equivalent to the
+origin. This result is also confirmed by the first equations.
+
+The solution of the other equations leads to the following **IJtoLS**-matrix:
+
+$$
+IJtoLS = \begin{pmatrix} 50 & 0 & 50 \\ 0 & -50 & 300 \\ 0 & 0 & 1 \end{pmatrix}
+$$
+
+In the event that a R(A)S basis was used, just the left and anterior axis of
+the anatomical space are flipped, and the image coordinate system appears in
+the same way as in the L\(P\)S case.
+
+![IJtoRS](https://github.com/Slicer/Slicer/releases/download/docs-resources/IJtoRS.png){.bg-warning w=550px align=center}
```

Comment:
```suggestion
![IJtoRS](https://github.com/Slicer/Slicer/releases/download/docs-resources/coordinate_systems_IJtoRS.png){w=550px align=center}
```

Reply:
Done in [0b7a79e](https://github.com/Slicer/Slicer/commit/0b7a79ed1b791335814ae637ee267ba8edf3750b).